front-end/src/player/PlayerQuestionnaireForm.tsx

```diff
@@ -48,42 +48,92 @@ export default function PlayerQuestionnaireForm(
   let maxAnswer = 40;
 
   const questionnaireInputs = (
-    <div className="questionnaireInputs">
```

Comment:
This is where the scrolling is removed because the questionnaireInputs class was removed from the questionnaireInputs div